R/orderly.R

```diff
@@ -74,6 +80,6 @@ orderly_new <- function(name, template = NULL, force = FALSE,
   }
 
   fs::dir_create(dest)
-  writeLines(contents, file.path(dest, "orderly.R"))
-  cli::cli_alert_success("Created 'src/{name}/orderly.R'")
+  writeLines(contents, file.path(dest, orderly_file_names[[1]]))
+  cli::cli_alert_success("Created 'src/{name}/{orderly_file_names[[1]]}'")
```

Comment:
It's not obvious from looking at these two lines which alternative is actually implemented.
Assuming we decide to make the new filename the blessed option, I would just make this `src/{name}/{name}.R`.